src/prothon/skills/prothon-tech-researcher/references/templates.md

```
@@ -0,0 +1,172 @@
+# Skill File Templates
+
+### Technology template
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)